apps/nextra/pages/en/build/smart-contracts/digital-asset.mdx

```diff
@@ -2,416 +2,321 @@
     collection::create_fixed_collection(
-      creator,
-      utf8(b"My Collection Description"),
-      max_supply,
-      utf8(b"My Collection"),
-      royalty,
-      utf8(b"https://mycollection.com"),
+        creator,
+        "My Collection Description",
+        max_supply,
+        "My Collection",
+        royalty,
+        "https://mycollection.com",
     );
-  }
 }
 ```
 
-2. Unlimited supply. Current supply is still tracked but there's no maximum
-enforced.
-
-```move
-module 0x42::example {
-  use aptos_token_objects::collection;
-  use std::string::utf8;
+1. For a `Collection` with unlimited supply you can use `collection::create_unlimited_collection`:
 
-  public entry fun create_collection(creator: &signer) {
+```move filename="Example.move"
+public entry fun create_collection(creator: &signer) {
     collection::create_unlimited_collection(
-      creator,
-      utf8(b"My Collection Description"),
-      utf8(b"My Collection"),
-      royalty,
-      utf8(b"https://mycollection.com"),
+        creator,
+        "My Collection Description",
+        "My Collection",
+        royalty,
+        "https://mycollection.com",
     );
-  }
 }
 ```
 
-Note that both track the current supply. Maximum supply cannot be changed after
-the collection is created, and a collection cannot be converted from unlimited
-to fixed supply or vice versa.
+<Callout type="warning">
+A `Collection`'s maximum supply cannot be changed after creation.
+</Callout>
 
-A collection has the following attributes:
+`Collection`s have the following fields:
 
-- Collection name - unique within each account. This means a single creator
-account cannot create more than one collection with the same name.
-- Description - modifiable with a `MutatorRef` and smaller than 2048 characters
-- URI length - modifiable with a `MutatorRef` and smaller than 512 characters
-- Royalty - specifies how many % of the sale price goes to the creator of the
-collection. This can be changed with a `MutatorRef` generated by the Royalty
-module.
+- Description - an optional string smaller than 2048 characters (modifiable with a `MutatorRef`).
+- Name - a required string to identify the `Collection`. The name must be unique within each account. That means a single creator account cannot create more than one `Collection` with the same name.
+- Royalty - an optional number indicating what % of the sale price goes to the creator of the `Collection`. This can be changed with a `MutatorRef` generated by the [Royalty module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/royalty.move) (an extension for the DA standard. See example usage in [`aptos_token.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/aptos_token.move))
+- URI length - an optional string that is smaller than 512 characters which links to relevant content for the `Collection` (modifiable with a `MutatorRef`).
```

Comment:
Let's move this to right below the `## Collections` header

```suggestion
| **Field** | **Description** |
|-----------|-----------------|
| **Description** | An optional string smaller than 2048 characters (modifiable with a `MutatorRef`). |
| **Name** | A required string to identify the `Collection`. The name must be unique within each account. That means a single creator account cannot create more than one `Collection` with the same name. |
| **Royalty** | An optional number indicating what % of the sale price goes to the creator of the `Collection`. This can be changed with a `MutatorRef` generated by the [Royalty module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/royalty.move) (an extension for the DA standard. See example usage in [`aptos_token.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/aptos_token.move)). |
| **URI length** | An optional string that is smaller than 512 characters which links to relevant content for the `Collection` (modifiable with a `MutatorRef`). |

```